packages/thirdweb/src/chains/types.ts

```diff
@@ -26,6 +26,12 @@ export type ChainOptions = {
     increaseZeroByteCount?: boolean;
   };
   faucets?: Array<string>;
+  fees?: {
+    /**
+     * Whether to force legacy transactions (pre EIP-1559) for this chain
+     */
+    forceLegacyTransactions?: boolean;
```

Comment:
Don't like the boolean because its inflexible, a `type: "legacy"` might be better (and matches viem/ox APIs)

Reply:
Might also want this on the transaction level if the user is providing it

Reply:
agreed on both counts. So something like `feeType: "legacy" | "eip1559"`?